.tekton/tasks/sync-argocd-app.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: app-sync
+spec:
+  description: >-
+    This task is for syncing the staging cluster argocd application
+  workspaces:
+   - name: argocd-config
+   - name: infra-deployment-git
+  steps:
+    - name: argocd-app-sync
+      image: quay.io/devtools_gitops/test_image:4.0.5
+      workingDir: $(workspaces.argocd-config.path)
+      script: |
+        #!/usr/bin/env python3
+        import subprocess
+        import re
+        import os
+
+        cmd = "pip3 install pyyaml"
+        output = subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = output.communicate()
+
+        # Load applications file
+        with open('$(workspaces.infra-deployment-git.path)/staging-app.yaml') as f:
+          import yaml
```

Comment:
Put the import statement at the top with the others.

Reply:
Since the image has `python3` but it does not have module for `yaml`. If you put at the top with others it will throw error
```
ModuleNotFoundError: No module named 'yaml'
```
And thats why I am installing `pyyaml` first and then `import yaml`